src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -208,6 +211,34 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         });
     }
 
+    Control::TermControl TermControl::AttachContent(Control::ControlInteractivity content, const Microsoft::Terminal::Control::IKeyBindings& keyBindings)
+    {
+        const auto term{ winrt::make_self<TermControl>(content) };
+        term->_AttachDxgiSwapChainToXaml(reinterpret_cast<HANDLE>(term->_core.SwapChainHandle()));
+        content.Reparent(keyBindings);
+
+        // Initialize the terminal only once the swapchainpanel is loaded - that
+        //      way, we'll be able to query the real pixel size it got on layout
+        auto r = term->SwapChainPanel().LayoutUpdated(winrt::auto_revoke, [term](auto /*s*/, auto /*e*/) {
+            // Replace the normal initialize routine with one that will allow up
+            // to complete initialization even though the Core was already
+            // initialized.
+            if (term->_InitializeTerminal(true))

```

Comment:
Can you add a comment before `true`?

```suggestion
            if (term->_InitializeTerminal(/* reattach */ true))
```

Reply:
I went an extra step

Reply:
I really really miss named arguments from python